index.bs

```diff
@@ -691,21 +691,114 @@ An {{MLOperand}} represents an intermediary graph being constructed as a result
 For instance, an {{MLOperand}} may represent a constant feeding to an operation or the result from combining multiple constants together into an operation. See also [[#programming-model]].
 
 <script type=idl>
+enum MLOperandKind {"constant", "input", "output"};
 [SecureContext, Exposed=(Window, DedicatedWorker)]
 interface MLOperand {};
 </script>
 
-See also [[#security-new-ops]]
+{{MLOperand}} has the following internal slots:
```

Comment:
To understand the picture: do you mean there would be an imaginary graph whose edge is copied to (belongs to), or just referenced by an MLOperand object as an internal slot?
Would MLGraph contain the whole of that graph as an internal slot and its elements referenced from MLOperands?

Reply:
Any input operand can also be an output depending on the operation taking or producing it. Maybe only a constant operand can't be an output, but it can also be an input. I'm not quite following the intent of this operand kind internal slot. What does it accomplish? If I were to implement it, I wouldn't worry about operand state. It's not significant.

Reply:
Removed [[kind]].